canvas_modules/common-canvas/src/palette/palette.scss

```diff
@@ -182,11 +183,12 @@ $palette-border-color: $ui-03;
 	cursor: pointer;
 	border-width: 0 0 0 $palette-border-width;
 	height: fit-content;
-	min-height: 46px;
+	max-height: 46px;
 	align-items: center;
 	justify-content: space-between;
 	display: flex;
-	border-bottom: $palette-border-width solid $palette-border-color;
+	margin-left: 0;
+	//border-bottom: $palette-border-width solid $palette-border-color;
 }
 
 .palette-flyout-category-caret {
```

Comment:
I think the palette-flyout-category-caret and palette-flyout-category-caret-closed  rulesets below, can be completely removed - right?